src/App.module.scss

```diff
@@ -0,0 +1,3 @@
+.main {
+  font-family: 'Mont', sans-serif;
```

Comment:
A small point I missed. Is it better to add this to normalize or to main scss. After all, this font is also needed in the header and footer.